modules/thunder_article/thunder_article.module

```diff
@@ -43,16 +43,11 @@ function _thunder_article_form_alter_helper(array &$form, FormStateInterface $fo
 function thunder_article_menu_local_tasks_alter(&$data, $route_name) {
   foreach ($data['tabs'] as $key => $tab) {
     foreach ($tab as $task => $link) {
-      if ($task === 'views_view:view.scheduler_scheduled_content.overview') {
+      $tabs = ['views_view:view.scheduler_scheduled_content.overview', 'thunder_article.content_view:thunder_article.scheduler'];
+      if (in_array($task, $tabs, TRUE)) {
         $config = \Drupal::service('config.factory')->get('thunder_article.settings');
```

Comment:
could we get this line, as well as line 46, in front of the foreaches?

Reply:
So the reason I didn't do that is that that every local_task_alter will do a query. This way we only do 1 query when we need to